### PR TITLE
Tweaks

### PR DESCRIPTION
**Pull request by benhaynes**

### CODE REVIEW

File: src/interfaces/file/file.vue [hunk 1]
```diff
@@ -160,6 +163,7 @@ export default defineComponent({
 
 		const imageThumbnail = computed(() => {
 			if (file.value === null) return null;
+			if (file.value.type.includes('svg')) return file.value.data.full_url;
```
Comment (rijkvanzanten) [inline src/interfaces/file/file.vue:166]: Use `asset_url` instead

File: src/modules/collections/routes/browse/browse.vue [hunk 2]
```diff
@@ -112,6 +107,26 @@
 			:filters.sync="filters"
 			:search-query.sync="searchQuery"
 		/>
+
+		<template #drawer>
+			<drawer-detail icon="info_outline" :title="$t('information')" close>
+				<v-detail>
+					<template #title>{{ $t('Need Help?') }}</template>
```
Comment (rijkvanzanten) [inline src/modules/collections/routes/browse/browse.vue:114]: Wrong translation here

```suggestion
					<template #title>{{ $t('need_help') }}</template>
```

also make sure to add this to en-us/index.json

File: src/modules/files/routes/browse/browse.vue [hunk 3]
```diff
@@ -7,6 +7,9 @@
 		</template>
 
 		<template #drawer>
+			<drawer-detail icon="info_outline" :title="$t('information')" close>
+				[TK]
```
Comment (rijkvanzanten) [inline src/modules/files/routes/browse/browse.vue:11]: Can we add the real content here? If we don't have it, lets not include it in this PR yet

File: src/modules/files/routes/detail/components/file-info-drawer-detail.vue [hunk 4]
```diff
@@ -68,7 +68,7 @@ export default defineComponent({
 		const size = computed(() => {
 			if (!props.filesize) return null;
 
-			return prettyBytes(props.filesize, { locale: i18n.locale.split('-')[0] });
+			return bytes(props.filesize, { decimalPlaces: 2, unitSeparator: ' ' }); // { locale: i18n.locale.split('-')[0] }
```
Comment (rijkvanzanten) [inline src/modules/files/routes/detail/components/file-info-drawer-detail.vue:71]: Remove old locale flag commented out

(Too bad this one doesn't support locales btw, now all us Dutchies have to watch the horror that's using a dot for the decimal place `12.5` 😱 )

File: src/modules/users/routes/browse/browse.vue [hunk 5]
```diff
@@ -7,6 +7,9 @@
 		</template>
 
 		<template #drawer>
+			<drawer-detail icon="info_outline" :title="$t('information')" close>
+				[TK]
```
Comment (rijkvanzanten) [inline src/modules/users/routes/browse/browse.vue:11]: "

File: src/modules/files/routes/detail/components/file-info-drawer-detail.vue (hunk 4 above)
Comment (benhaynes) [reply src/modules/files/routes/detail/components/file-info-drawer-detail.vue:71]: I'm pretty sure it does support i18n... but I didn't enable it. Might need other files/dependencies?

Comment (benhaynes) [reply src/modules/files/routes/detail/components/file-info-drawer-detail.vue:71]: Did you change this? It was already done...

**Pull request merged**